docs/topics/development/debugging.rst

```diff
@@ -90,6 +88,10 @@ If clicked, it looks like:
 
 .. image:: /screenshots/django-debug-toolbar-expanded.png
 
+You must know that using the Django Debug Toolbar will slow the website quite a
+lot. You can mitigate this by deselecting the checkbox next to the ``SQL``
+panel.
```

Comment:
Could we add something like if you want the DjDT you can turn off CSP in about:config? Although then we end with developers not having CSP enabled and missing changes that break CSP. So probably not.